_sass/_docs.scss

```diff
@@ -1,3 +1,92 @@
+
+.sidebar {
+    top: 70px;
+    left:0;
+    width: $sidebar-width; 
+    height: 100vh;
+    overflow-y: auto;
+    overflow-x: hidden;
+    flex-grow: 2;
+    padding: 30px 30px 0px 10px;
+    position: sticky;
+    background-color: #e6e4e4;
+    text-align: right;
+    @media (max-width: $desktop-width) {
+        display: none;
+    }
+}
+
+.sidebar-content {
+    display: inline-block;
+    text-align: left;
+    width: 225px;
+}
```

Comment:
To reiterate my previous comment, many of the styles specified here could be abstracted away with a [flex container](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox).